lively.collab/comments/commentBrowser.js

```diff
@@ -25,9 +25,7 @@ export class CommentBrowser extends Window {
   }
 
   static async addCommentForMorph (comment, morph) {
-    if (CommentBrowser.isOpen()) {
-      await instance.addCommentForMorph(comment, morph);
-    }
+    await instance.addCommentForMorph(comment, morph);
```

Comment:
Will result in an error when the browser is not opened yet won't it?

Reply:
We do not think it is possible to reach this without opening the browser.